float64 epaule_av
float64 epaule_ar
